hooks/pre-command

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+aws_account_id="697149045717"
+hash=$(echo -n "$(echo "${BUILDKITE_REPO}" | awk -F'[:.]' '{ printf $3 }')/${BUILDKITE_PIPELINE_SLUG}" | sha256sum | cut -c1-55)
```

Comment:
Can you add a comment about the rationale?

Reply:
https://github.com/elastic/oblt-aws-auth-buildkite-plugin/pull/1/commits/341685cd11edfff482a611144f3354951422610e